packages/compass-components/src/components/resize-handle.tsx

```diff
@@ -0,0 +1,168 @@
+/** @jsx jsx */
+import { css, jsx } from '@emotion/react';
+import React, { useState } from 'react';
+
+const baseResizerStyles = css({
+  position: 'absolute',
+  background: 'transparent',
+  padding: 0,
+  transition: 'opacity 250ms ease',
+  transitionDelay: '0ms',
+  backgroundColor: '#019EE2',
+  opacity: '0',
+  outline: 'none',
+  zIndex: 100,
+  ':focus': {
+    opacity: 1,
+  },
+  ':hover': {
+    transitionDelay: '250ms',
+    opacity: 1,
+  },
+  WebkitAppearance: 'none',
+  '::-webkit-slider-thumb': {
+    WebkitAppearance: 'none',
+  },
+  '::-ms-track': {
+    background: 'none',
+    borderColor: 'none',
+    color: 'none',
+  },
+});
+
+const verticalResizerStyle = css({
+  width: '4px !important',
```

Comment:
Without it one of the bootstrap classes takes over, we could override that selector with one that doesn't use !important, but we'd need to do some wrapping of the input. Not much of a fan of using `!important` but I think this is a pretty scoped usage of it.

![Screen Shot 2021-09-29 at 11 57 47 AM](https://user-images.githubusercontent.com/1791149/135307309-4b97000b-f910-4fbf-9adf-1a8271e692c2.png)

Reply:
Hmmm, got it, yeah it's very scoped, I just somehow didn't notice this issue when running locally, thanks for explaining! (As a side-note it might be something misconfigred in the build, might be worth looking into, it's a bit weird that bootstrap styles are applied after leafygreen ones)